app.py

```diff
@@ -1,641 +1,44 @@
 import os
-import pymongo
-import json
+from flask import Flask
+from flask_talisman import Talisman
 
-from pymongo import MongoClient
-from bson import ObjectId
-from bson.json_util import dumps
 
-from datetime import datetime
-from flask import Flask, redirect, render_template, request, session, url_for
-from passlib.hash  import sha256_crypt
+from blueprints.login_logout import login_logout
+from blueprints.home import home
+from blueprints.api import api
+from blueprints.filters import filters
+from blueprints.profiles import profiles
+from blueprints.add_drink import add_drink
+from blueprints.comments import comment
+from blueprints.update import update
+from blueprints.delete import delete
+from blueprints.search import searchs
 
 
 app = Flask(__name__)
 
-mongo_uri = os.environ.get('MONGO_URI')
-DBS_NAME = os.environ.get('DBS_NAME')
-app.secret_key = os.environ.get('SECRET_KEY')
+app.secret_key = "asdfsdfs"
```

Comment:
We don't do that here :)
Put all confidential info into a `.env` file and read with Dotenv https://github.com/theskumar/python-dotenv